### PR TITLE
Guess high atop the stack

### PR DESCRIPTION
**Pull request by jondequinor**
If we get this in, I will be happy with record transmitting for now

### CODE REVIEW

**Review by sondreso — APPROVED** [review]
💯

**Comment by jondequinor** [comment]
Yeah need to fix them tomorrow. Sorry for making it ready. It was not.

**Review by sondreso — APPROVED** [review]
💯 🥇

File: ert3/config/_stages_config.py [hunk 1]
```diff
@@ -36,29 +52,25 @@ class Config:
...
+    mime: str = ""
+
+    _ensure_mime = validator("mime", allow_reuse=True)(_ensure_mime)
 
 
 class OutputRecord(_StagesConfig):
     record: str
     location: str
-    mime: Optional[str]
+    mime: str = ""
```
Comment (DanSava) [inline ert3/config/_stages_config.py:63]: Why does the default need to be `""` and not `_DEFAULT_RECORD_MIME_TYPE` ?

File: ert3/config/_stages_config.py [hunk 2]
```diff
@@ -36,29 +52,25 @@ class Config:
 class InputRecord(_StagesConfig):
     record: str
     location: str
-    mime: Optional[str]
+    mime: str = ""
```
Comment (DanSava) [inline ert3/config/_stages_config.py:55]: Why does the default need to be `""` and not `_DEFAULT_RECORD_MIME_TYPE` ?

File: ert3/config/_stages_config.py [hunk 3]
```diff
@@ -36,29 +52,25 @@ class Config:
...
+
+    _ensure_mime = validator("mime", allow_reuse=True)(_ensure_mime)
 
 
 class TransportableCommand(_StagesConfig):
     name: str
     location: FilePath
     mime: str = ""
```
Comment (DanSava) [inline ert3/config/_stages_config.py:71]: Why does the default need to be `""` and not `_DEFAULT_CMD_MIME_TYPE` ?

**Pull request merged**